Refactor X resolution (#4622)
cocotb/cocotb: docs/source/newsfragments/4622.feature.2.rst

> Added :func:`cocotb.handle.set_resolve_x` for changing the non-``0``/``1`` value resolution behavior at runtime, which was previously limited to begin set with :envvar:`COCOTB_RESOLVE_X`.

Reviewer: ```suggestion
Added :func:`cocotb.handle.set_resolve_x` for changing the non-``0``/``1`` value resolution behavior at runtime (overriding :envvar:`COCOTB_RESOLVE_X` if set).
```

(Not quite sure what you wanted to say with "begin set".)
